Login screen(diego)

unhumanosinmas opened the pull request:
Se implementaron TextInputs Perzonalizados y marcados como componentes

plancer0 committed:
Integracion de fuentes y temas

plancer0 committed:
Button Component

unhumanosinmas committed:
Componente TextInput

creacion de componente TextInput Personalizado

unhumanosinmas committed:
Merge remote-tracking branch 'origin/peter' into diego

unhumanosinmas committed:
Login Preliminar

Login de Referencia

plancer0 committed:
Inputs

plancer0 committed:
SVG Icons and register form

plancer0 committed:
Home page

unhumanosinmas committed:
Merge branch 'register_screen(peter)' into login_screen(diego)

unhumanosinmas committed:
3 views complete

se realizo adaptó la view del Login

unhumanosinmas committed:
Fixed Views

Color de los placeholders módificados

plancer0 committed:
Correcion de primera pull request

unhumanosinmas committed:
FIxed Login InputText

se arreglaron los imputs

unhumanosinmas committed:
Fixed InputText Register

Se hizo uso del componente customizado del InputText

unhumanosinmas commented:
Se me olvido agregar que también va la vista del login dentro de las cosas que hice

GibranPolonsky commented on src/components/buttons/buttons.tsx:
```diff
@@ -0,0 +1,14 @@
+import {View, Text, TouchableOpacity} from 'react-native';
+import buttonStyle from './buttons.style';
+
+const Button = ({title}: {title: string}) => {
```
aqui podes pasar el on press que es requerido y otros props necesarios en el component

GibranPolonsky commented on src/components/textInput/textInput.tsx:
```diff
@@ -0,0 +1,20 @@
+import theme from '../theme'
+
+
+const CustomTI = ({placeholder, secureTextEntry}: {placeholder: string, secureTextEntry: boolean} ) => {
```
seria bueno pasar el value y el onChange aqui

GibranPolonsky commented on src/views/Home/home.tsx:
```diff
@@ -0,0 +1,30 @@
+
+const HomeView = () => {
+  return (
+    <View style={homeStyles.homeContainer}>
```
Tratemos la manera de siempre usar SafeArea

unhumanosinmas replied:
### Ya está resuelto
El safe área está incluida en el App para que así aplique a todas las vistas


![image](https://user-images.githubusercontent.com/104159636/173197503-87f105bb-5a14-4d0c-b02f-6e909c421947.png)

GibranPolonsky commented on src/views/Register/register.tsx:
```diff
@@ -0,0 +1,57 @@
+      placeholder="Name"
+      secureTextEntry={false}
+      />
+      <CustomTI
```
ha este no le tenes porque cambiar el nombre, le podes llamar TextInput

plancer0 committed:
Identación de código

Se idento el codigo y se empezo a implementar la navegacion

plancer0 committed:
Merge remote-tracking branch 'origin/login_screen(diego)' into register_screen(peter)

plancer0 committed:
Arreglo de rutas

Movi la carpeta utils a src y actualice rutas

plancer0 committed:
Arreglo de redundancias y errores

Saque codigo de estilo que no servia o que provocaban redundancias y arregle un error en los componentes, arregle mucha parte de los componentes boton e input

plancer0 committed:
Navigation

Se implemento la navegacion el proyecto

unhumanosinmas committed:
Background color undefined bug (Fixed)

Se solucionaron problemas con el color de fondo de las vistas

plancer0 committed:
Iconos y estilo

plancer0 committed:
Login

Se trato de implementlar el login

plancer0 committed:
Limpiando proyecto de intento de login

plancer0 committed:
Arreglo de errores con estilos

plancer0 committed:
Merge remote-tracking branch 'origin/login_screen(diego)' into register_screen(peter)

unhumanosinmas committed:
Servicios para Redux

Se crearon lo servicios para usar redux

plancer0 committed:
Login with redux

se arreglaron errores de estilos, propiedades de los componentes y se implemento un sistema para iniciar sesión, aunque es un poco básico y se puede mejorar

unhumanosinmas committed:
Merge remote-tracking branch 'origin/register_screen(peter)' into login_screen(diego)

unhumanosinmas committed:
Merge remote-tracking branch 'origin/register_screen(peter)' into login_screen(diego)

GibranPolonsky commented on src/reducers/auth.ts:
```diff
@@ -0,0 +1,28 @@
+      return {
+        ...state,
+        isLoggedIn: true,
+        user: payload.user,
```
A Este campo me gustaría que llamaras toen, en vez de user, porque realmente es un token y no un objeto de usuario

GibranPolonsky commented:
Por todo lo demás yo lo veo bastante bien, el nombre lo podes cambiar después, este PR está aprobado!
